### PR TITLE
Fix registration breadcrumb

### PR DESCRIPTION
**Pull request by RodrigoLeiteF**
## What issue does this PR close
Closes N/A

## Changes Proposed ( a list of new changes introduced by this PR)
Changes the text for the step breadcrumb when user advances to step 2

## How to test ( a list of instructions on how to test this PR)
1. Go to the registration page and create an account

### CODE REVIEW

**Comment by amiskov** [comment]
It looks like there are some issues with the design. Here's the 2nd step from `master` branch:
image
And here is the current branch:
image

**Comment by RodrigoLeiteF** [comment]
Ah, I didn't realize we already had that in our current flow, and since it's not on Figma I assumed it would look the same for both steps. I'm on it

**Comment by RodrigoLeiteF** [comment]
Submitted a fix that changes it to look like this:
image

According to Helton the placement is fine, since it stays consistent with the previous step on the flow.

**Comment by amiskov** [comment]
> since it's not on Figma I assumed it would look the same for both steps. I'm on it

It was in Figma, I was implementing [this design](https://github.com/cambiatus/frontend/issues/151#issuecomment-614108671) some time ago, but it's lost now :(

File: src/elm/Page/Register.elm [hunk 1]
```diff
@@ -295,8 +295,19 @@ viewCreateAccount translators model =
 
                 _ ->
                     div [] []
+
+        backgroundColor =
+            case model.step of
+                1 ->
+                    "bg-white"
```
Comment (amiskov) [inline src/elm/Page/Register.elm:302]: This case is already covered by `_`.

**Review by amiskov — CHANGES_REQUESTED** [review]
All looks good, I have only one little suggestion.

File: src/elm/Page/Register.elm (hunk 1 above)
Comment (RodrigoLeiteF) [reply src/elm/Page/Register.elm:302]: Fair enough :p

**Review by amiskov — APPROVED** [review]

**Pull request merged**